synapse/handlers/federation_event.py

```diff
@@ -1850,7 +1895,7 @@ async def persist_events_and_notify(
                     store=self._store,
                     room_id=room_id,
                     event_and_contexts=batch,
-                    backfilled=backfilled,
+                    inhibit_push_notifications=inhibit_push_notifications,
```

Comment:
doesn't this need to pass through *all* the keyword args?

Reply:
👍 I think so. Should match `_serialize_payload`.

It's a loose connection and I was having trouble typing it correctly, https://github.com/matrix-org/synapse/pull/11396#discussion_r752929555, to have the linter yell at me for this.